src/v/kafka/client/client.cc

```diff
@@ -191,7 +191,7 @@ ss::future<> client::mitigate_error(std::exception_ptr ex) {
                   vlog(kclog.debug, "topic_error: {}", ex);
                   return _wait_or_start_update_metadata();
               default:
-                  vlog(kclog.warn, "topic_error: {}", ex);
+                  vlog(kclog.info, "topic_error: {}", ex);
```

Comment:
This was originally "warn" because it's processing an `ec` without explicitly handling it. The goal is to make it visible, in case there's a bug.

Perhaps `throttling_quota_exceeded` can be explicitly handled with a debug log, and then leave the default with warn log?

Reply:
This change was an accident.